docs/user-guide.md

```diff
@@ -408,6 +408,9 @@ spec:
           value: "2"
         - name: edns0
 ```
+### Overriding args and commands
```

Comment:
Capital case of the first letter of each word.

Reply:
Ah, ok, I was trying to aim for the name in the yaml/json. Changed.